system/filesystem/check-tail.yml

```diff
@@ -0,0 +1,105 @@
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: check-tail
+  namespace: default
+spec:
+  command: check-tail.rb -f {{ .labels.check_tail_fail | default /var/log/messages }} -P {{ .labels.check_tail_pattern | default error }} -l {{ .labels.check_tail_lines | default 10 }}
```

Comment:
tail_fail made me giggle 🤣

Reply:
Hah - that was a typo! It should be file. I'll update this.